### PR TITLE
fix php7

### PR DESCRIPTION
**Pull request by hmartin**
#24

### CODE REVIEW

File: src/Type/JsonDocumentType.php [hunk 1]
```diff
@@ -109,8 +109,8 @@ public function convertToDatabaseValue($value, AbstractPlatform $platform)
      */
     public function convertToPHPValue($value, AbstractPlatform $platform)
     {
-        if (null === $value) {
-            return;
+        if (null === $value || $value === '') {
+           return array();
```
Comment (dunglas) [inline src/Type/JsonDocumentType.php]: `null` is valid JSON isn't it? IMO we should return `null` instead of `[]` in this case.

Comment (hmartin) [reply src/Type/JsonDocumentType.php]: https://3v4l.org/TilkM
json_decode(null) throw an exception

Comment (dunglas) [reply src/Type/JsonDocumentType.php]: It will not be called right?

**Comment by hmartin** [comment]
I did same as https://github.com/doctrine/dbal/blob/master/lib/Doctrine/DBAL/Types/JsonArrayType.php
But you're rigth!

**Comment by dunglas** [comment]
Thanks @hmartin

**Pull request merged**